staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -239,7 +269,9 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 
 	status := &v1.LoadBalancerStatus{}
 	status.Ingress = []v1.LoadBalancerIngress{{IP: updatedFwdRule.IPAddress}}
-	return status, nil
+	klog.V(2).Infof("ensureInternalLoadBalancer(%v): Adding ILB status annotations to %q", loadBalancerName, ILBFinalizerV1)
```

Comment:
Should this include service/nsname instead of ILBFinalizerV1?

Reply:
Good catch, thanks!